django/middleware/csrf.py

```diff
@@ -27,6 +27,8 @@
 REASON_INSECURE_REFERER = "Referer checking failed - Referer is insecure while host is secure."
 
 CSRF_KEY_LENGTH = 32
+CSRF_TOKEN_LENGTH = CSRF_KEY_LENGTH + CSRF_KEY_LENGTH
```

Comment:
I thought it was clearer that it's key-length + pad-length this way, but I may be wrong.